.travis.yml

```diff
@@ -8,6 +8,8 @@ sudo: false
 before_install:
    - export PATH=$HOME/gopath/bin:$PATH
 
+install: "pip install --user -r requirements.txt"
```

Comment:
actually, the travis containers are running ubuntu, so that would be `apt-get` :)
ie: `apt-get install python-cffi`